python/lsst/ts/envsensors/socket_server.py

```diff
@@ -45,8 +46,9 @@ class SocketServer:
 
     valid_simulation_modes = (0, 1)
 
-    def __init__(self, port, simulation_mode=0):
-        self.port = port
+    def __init__(self, port: int, simulation_mode: int = 0) -> None:
+        self.host: str = "0.0.0.0"  # always read on all interfaces
```

Comment:
I gently suggest not specifying host at all, unless you think you may need to limit which interfaces you will read on. That matches the convention used by asyncio (at least) and thus OneClientServer. (Or, equivalently, host=None).

Reply:
Done